gooddata-sdk/gooddata_sdk/compute/model/execution.py

```diff
@@ -259,15 +261,24 @@ def read_result(self, limit: Union[int, list[int]], offset: Union[None, int, lis
         # this makes sure that offset gets defaulted to start of result
         _offset = [0 for _ in _limit] if _limit is not None and _offset is None else _offset
 
-        return ExecutionResult(
-            self._actions_api.retrieve_result(
-                workspace_id=self._workspace_id,
-                result_id=self.result_id,
-                offset=_offset,
-                limit=_limit,
-                _check_return_type=False,
-            )
+        execution_result, _, http_headers = self._actions_api.retrieve_result(
+            workspace_id=self._workspace_id,
+            result_id=self.result_id,
+            offset=_offset,
+            limit=_limit,
+            _check_return_type=False,
+            _return_http_data_only=False,
         )
+        custom_headers = self._api_client.custom_headers
+        if "X-GDC-TRACE-ID" in custom_headers and "X-GDC-TRACE-ID" in http_headers:
+            logging.getLogger(__name__).info(
```

Comment:
Move `logging.getLogger(__name__)` to package variable so that others can benefit from it, i.e.
```
logger = logging.getLogger(__name__)
```

Reply:
fixed